mono/utils/mono-threads-coop.c

```diff
@@ -273,13 +320,13 @@ mono_threads_core_end_global_suspend (void)
 void*
 mono_threads_enter_gc_unsafe_region (void)
 {
-	return mono_threads_reset_blocking_start ();
+	return mono_threads_reset_blocking_start (0);
 }
 
 void
 mono_threads_exit_gc_unsafe_region (void *regions_cookie)
 {
-	mono_threads_reset_blocking_end (regions_cookie);
+	mono_threads_reset_blocking_end (regions_cookie, 0);
```

Comment:
Yes, we should never pass 0 when expecting a pointer, it makes the code worse to read than needed.